boards/arm/nrf9160_pca20035/Kconfig.board

```diff
@@ -0,0 +1,25 @@
+# Kconfig - nRF91 PCA20035 board configuration
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+if SOC_NRF9160_SICA
+
+config BOARD_NRF9160_PCA20035_V0_2_2
```

Comment:
See my comment on the Kconfig.board file for `nrf52840_pca20035`.

Reply:
Added comment above